relayer/src/chain/cosmos.rs

```diff
@@ -308,7 +308,7 @@ impl CosmosSdkChain {
         };
 
         let mut tx_bytes = Vec::new();
-        prost::Message::encode(&tx_raw, &mut tx_bytes).unwrap();
+        prost::Message::encode(&tx_raw, &mut tx_bytes).map_err(|e| Error::protobuf_encode(String::from("Transaction"), e))?;
```

Comment:
I guess it's very tricky if not impossible to reproduce this error? I'd be curious to see how it propagates up the call stack and at which level do we retry (it should be ideally as low-level as possible).

Reply:
It's not clear to me how that can even happen, would have to dig through the `prost` code to figure it out. Regardless, we wan't to avoid crashing Hermes if that happens and therefore have to bubble up the error here. We don't act on this error in any specific way so this will just trigger retries in the workers, and very likely fail repeatedly so it will be seen in the logs. As we won't ever get to submitting a tx, it won't drain the wallet either which makes the current solution acceptable for now. What do you think?